src/module-elasticsuite-core/etc/adminhtml/di.xml

```diff
@@ -22,7 +22,19 @@
         <arguments>
             <argument name="messages" xsi:type="array">
                 <item name="elasticsuite" xsi:type="string">Smile\ElasticsuiteCore\Model\System\Message\NotificationAboutVersions</item>
-                <item name="elasticsuite_cluster_replicas_misconfig_warning" xsi:type="string">Smile\ElasticsuiteCore\Model\System\Message\WarningAboutClusterReplicasMisconfig</item>
+                <!-- Generic warning class -->
+                <item name="elasticsuite_cluster_generic_misconfig_warning" xsi:type="string">Smile\ElasticsuiteCore\Model\System\Message\GenericWarningAboutClusterMisconfig</item>
```

Comment:
Same goes as for the class name, "elasticsuite_generic_healthcheck_warning" would be better here to be consistent.